pkg/codegen/testing/test/program_driver.go

```diff
@@ -146,13 +146,6 @@ var PulumiPulumiProgramTests = []ProgramTest{
 		Directory:   "random-pet",
 		Description: "Random Pet",
 	},
-	{
-		Directory:   "aws-resource-options",
```

Comment:
What is this a duplicate of?

Reply:
Right it's covered by [TestGenerateProgramVersionSelection](https://github.com/pulumi/pulumi/blob/friel/enable-versioned-cache/pkg/codegen/go/gen_program_test.go#L73)